core-services/secrets/sd-sre-secrets/sd-sre-secrets.yaml

```diff
@@ -0,0 +1,61 @@
+# this is the Namespace in which your Secret will live
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Automation Secrets for SD SRE
+    openshift.io/display-name: SD SRE CI
+  name: sd-sre-secrets
+---
+# the Group of people who should be able to manage this Secret
+kind: Group
+apiVersion: v1
+metadata:
+  name: sd-sre-secrets-admins
+users:
+  # these names are GitHub usernames
+  # SREP Team lads
+  - jharrington22
+  - cblecker
+  - mwoodson
+  - jewzaam
+  # 
```

Comment:
We should get @dofinn in here

Reply:
Wasn't adding everyone right now. Just wanted to get this in.

Updated.

Reply:
Sho, just wanted to make sure we didn't exclude [people who were previously auth'd](https://github.com/openshift/ops-sop/blob/1e48b8bf8fbb2237a0348fbf97a206a29dcbd905/services/codecov.md#create-a-secret-in-the-ci-cluster)